lua/neogit/lib/buffer.lua

```diff
@@ -137,8 +137,9 @@ function Buffer.exists(name)
   return vim.fn.bufnr(name) ~= -1
 end
 
-function Buffer.create(config)
+function Buffer.create(config, options)
   local config = config or {}
+  local width = options.width or 50
```

Comment:
I don't think the default width should be a static value. Providing no width should behave like executing `vnew` without a width argument.

Reply:
yes, I think there is no need to add a configure about windows width. But add "vsplit" to usage section is necessary.

Reply:
Could you add a usage section about "vsplit".